addons/account/models/chart_template.py

```diff
@@ -1310,3 +1325,31 @@ def _load_translations(self, langs=None, companies=None, template_data=None):
                             break
 
         translation_importer.save(overwrite=False)
+
+    @api.model
+    def _update_reconciliation_models(self, company):
+        """
+        The write-off button default reconciliation model for Bill creation
+        requires tax ids, which need to be set after the default taxes are defined.
+        """
+        lines = self.env['account.reconcile.model'].search([
+            *self.env['account.reconcile.model']._check_company_domain(company),
+            ('counterpart_type', '=', 'purchase'),
+        ]).line_ids
```

Comment:
This should only be done for `reconcile_bill`
```suggestion
        lines = self.ref('reconcile_bill').line_ids
```